Gruntfile.coffee

```diff
@@ -586,6 +587,41 @@ module.exports = (grunt) ->
 				staticImagePath: '#{$wb-assets-path}'
 				output: "scss"
 
+		lintspaces:
+			all:
+				src: [
+						"*"
```

Comment:
Not sure how best to handle this kind of situation...

Trying to look for any files at the root of the repo - without inadvertently evaluating ignored (and possibly un-ignored untracked files) and without turning this into an unmanageable mess of explicit file inclusions/exclusions.

Current way is simply-written, but doesn't ignore anything.

Reply:
Wasn't able to get it working. So I came up with a small set of globbing patterns that seem to work decently. They'll appear in my next force push.